lighthouse-core/test/audits/speed-index-metric-test.js

```diff
@@ -74,7 +74,6 @@ describe('Performance: speed-index-metric audit', () => {
     const artifacts = mockArtifactsWithSpeedlineResult(SpeedlineResult);
 
     return Audit.audit(artifacts).then(response => {
-      assert.equal(response.displayValue, '845');
```

Comment:
i saw when you were debugging this..

looks like you nuked it because `.displayValue` is never explictly set?